### PR TITLE
change-background-color-of-code-block

### PR DESCRIPTION
**Pull request by farshidinanloo**
currently background color of block of code in light theme is white and the page background is also white so it's hard to distinguish between code block from other parts so I change it to be different.
this is currently how it looks like

and I just change the background color to be more readable:

### CODE REVIEW

**Comment by rrousselGit** [comment]
this change makes Dark mode quite a bit harder to read:

with this change:
Screenshot 2022-04-02 at 12 23 00

before:

Screenshot 2022-04-02 at 12 23 12

**Comment by farshidinanloo** [comment]
@rrousselGit 
that's right
I changed it to only have effects in light theme

**Comment by rrousselGit** [comment]
Much better :)

Also, it probably shouldn't apply the toe code snippet at the top of the home-page:

**Comment by farshidinanloo** [comment]
now home page code snippets have correct background color

File: website/src/components/CodeSnippet/index.tsx [hunk 1]
```diff
@@ -28,9 +29,11 @@ interface CodeSnippetProps {
   snippet: string;
 }
 
-export const CodeSnippet: React.FC<CodeSnippetProps> = ({ snippet, title }) => {
+export const CodeSnippet: React.FC<CodeSnippetProps> = ({ snippet, title}) => {
+  const {isDarkTheme} = useThemeContext()
+
   return (
-    <div className="snippet">
+    <div className={`snippet ${!isDarkTheme && 'whiteCodeBlock'}`}>
```
Comment (rrousselGit) [inline website/src/components/CodeSnippet/index.tsx:36]: This will lead to `snippet false` if the condition fails. That `false` probably shouldn't be here

Comment (rrousselGit) [reply website/src/components/CodeSnippet/index.tsx:36]: Also it appears that this condition is false during the initial rendering, even on white theme

See the deployed test. When refreshing the website, the snippet starts with a grey background.
Then changing to dark theme and back to light theme, the background is correctly white

**Comment by farshidinanloo** [comment]
sorry for the mistakes that second problem was because default theme is dark and then it figure out it should be light but useTheme apparently doesn't update the data, now I put in setTimeout in useEffect  it works but I hope is the correct way

**Comment by rrousselGit** [comment]
LGTM, thanks for your work 😄

**Pull request merged**